configs/mainnet.yaml

```diff
@@ -116,8 +116,6 @@ DEPOSIT_CONTRACT_ADDRESS: 0x00000000219ab540356cBB839Cbe05303d7705Fa
 GOSSIP_MAX_SIZE: 10485760
 # `2**10` (= 1024)
 MAX_REQUEST_BLOCKS: 1024
-# `2**8` (= 256)
-EPOCHS_PER_SUBNET_SUBSCRIPTION: 256
```

Comment:
I vague recall there was a discussion on this somewhere but I couldn't find it - does the rotation not provide any benefit?

Reply:
It's buried here https://github.com/ethereum/consensus-specs/pull/3623